src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+interface Tag {
+  label: string;
+  color: string;
+}
+
+interface DataCardProps {
+  title: string;
+  tags: Tag[];
+  description: string;
+  publishedAt: string;
+  version: string;
+  sourceUrl: string;
+  downloadUrl: string;
+}
+
+export default function DataCard({
+  title,
+  tags,
+  description,
+  publishedAt,
+  version,
+  sourceUrl,
+  downloadUrl,
```

Comment:
Instead of giving each card attribute to the component it would be a better approach to add the whole entity at once

```suggestion
  card: IDataCard 
```

With this approach if an IDataCard attribute is added or removed we won't need to update the interface (the componente prop), since we're given the whole entity instead of a list of attributes